src/components/Api.js

```diff
@@ -0,0 +1,168 @@
+class Api {
+  constructor(options) {
+    this._authorization = "fd5203a5-1f57-4428-b210-42e565daa250";
+  }
+
+  getUserInfo() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/users/me", {
```

Comment:
Needs correction: it's needed to save the base url in the constructor and reuse it in all the api methods